maestro-ios/src/main/java/ios/LocalIOSDevice.kt

```diff
@@ -56,7 +56,8 @@ class LocalIOSDevice(
                         Insight(
                             message = "Retrieving the hierarchy is taking longer than usual. This might be due to a " +
                                     "deep hierarchy in the current view. Please wait a bit more to complete the operation.",
-                            level = Insight.Level.WARNING
+                            level = Insight.Level.WARNING,
+                            visibility = Insight.Visibility.VISIBLE
```

Comment:
This is a bit confusing to me since visibility seems to only applies to studio and not the terminal output

Reply:
It is reflecting there its on diffferent PR. Request you to review that as well: https://github.com/mobile-dev-inc/maestro/pull/1274/files

Reply:
What I meant is that the visibility field only seems to control whether this is displayed in the studio banner. This is surprising since I would expect a field named `visibility` to apply in all cases.

Reply:
Ahhh right thats true. I'll do that